tunnel/intercept/svcpoll.go

```diff
@@ -101,21 +114,42 @@ func (self *ServiceListener) HandleServicesChange(eventType ziti.ServiceEventTyp
 func (self *ServiceListener) addService(svc *entities.Service) {
 	log := pfxlog.Logger()
 
+	svc.DialTimeout = 5 * time.Second
+
 	if stringz.Contains(svc.Permissions, "Dial") {
-		clientConfig := &entities.ServiceConfig{}
-		found, err := svc.GetConfigOfType(entities.ClientConfigV1, clientConfig)
+		interceptV1Config := &entities.InterceptV1Config{}
+		found, err := svc.GetConfigOfType(entities.InterceptV1, interceptV1Config)
+		if found {
+			svc.InterceptV1Config = interceptV1Config
+			if interceptV1Config.DialOptions != nil && interceptV1Config.DialOptions.ConnectTimeoutSeconds != nil {
+				svc.DialTimeout = time.Duration(*interceptV1Config.DialOptions.ConnectTimeoutSeconds) * time.Second
+			}
+		}
 
-		if found && err == nil {
-			svc.ClientConfig = clientConfig
-		} else if !found {
-			pfxlog.Logger().Debugf("no service config of type %v for service %v", entities.ClientConfigV1, svc.Name)
-		} else if err != nil {
-			pfxlog.Logger().WithError(err).Errorf("error decoding service config of type %v for service %v", entities.ClientConfigV1, svc.Name)
+		if err != nil {
+			logrus.WithError(err).Errorf("error decoding service config of type %v for service %v", entities.InterceptV1, svc.Name)
+		}
+
+		if !found {
+			clientConfig := &entities.ServiceConfig{}
+			found, err = svc.GetConfigOfType(entities.ClientConfigV1, clientConfig)
```

Comment:
Ahh. Disregard my previous question about client.v1. (I'm still looking for the place where intercept protocol is handled though)

Reply:
tproxy_linux lines 140 and 149